packages/spec/integration/redux/__tests__/develop.js

```diff
@@ -34,7 +34,20 @@ describe('redux developmet server', () => {
     const { page, getInnerText } = await createPage();
     await page.goto(url + '/increment', { waitUntil: 'networkidle2' });
 
-    const count = await getInnerText('output');
+    const count = await getInnerText('counter');
+
+    expect(count).toBe('1');
+
+    await page.close();
+  });
+
+  it('executes route-matching action creators also if a search query or hash is present', async () => {
```

Comment:
I'd personally like to wrap these `route-matching action creators` tests into a `describe` declaration, like. How do like it?

```js
describe('route-matching action creators')
  it('executes it when url contains search query or hash')
  it('executes it with fetch')
  it('provides it with expected "param" string')
  it('provides it with the expected  "match" object')
  it('provides it with the expected  "location" object')
```